Refresh.HttpServer/RequestContext.cs

```diff
@@ -1,8 +1,10 @@
 using System.Net;
+using NotEnoughLogs;
 
 namespace Refresh.HttpServer;
 
 public struct RequestContext
 {
     public HttpListenerRequest Request;
+    public LoggerContainer<RefreshContext>? Logger;
```

Comment:
```suggestion
    public LoggerContainer<RefreshContext> Logger;
```

Shouldn't be nullable